src/components/EventCalender/Yearly/YearlyEventCalender.tsx

```
@@ -210,27 +251,24 @@
+                data-testid={isExpanded ? 'expanded-event-list' : undefined}
               >
-                {expandedY === expandKey && renderedEvents}
+                {isExpanded && renderedEvents}
               </div>
+
               {eventsForDate.length > 0 ? (
                 <button
                   className={styles.btn__more}
                   onClick={() => toggleExpand(expandKey)}
                   data-testid={`expand-btn-${expandKey}`}
                 >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)